docs/source/destination_connectors/pinecone.rst

```diff
@@ -0,0 +1,74 @@
+S3
```

Comment:
title is incorrect (still s3)

Reply:
Agreed, addressed with [doc, comment and logging updates](https://github.com/Unstructured-IO/unstructured/pull/1774/commits/6d84efc018bb5a2d466c06caa56705e9531d2af7)